src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.ts

```diff
@@ -116,6 +121,7 @@ export class ContentTypeFieldsDropZoneComponent extends BaseComponent implements
         fields.forEach(field => {
```

Comment:
you might probably use filter here, fields.filter (field => this.isNewField(field)).forEach(field => ...);